release_tester/arangodb/installers/base.py

```diff
@@ -15,8 +15,11 @@
 import psutil
 from arangodb.instance import ArangodInstance
 from tools.asciiprint import print_progress as progress
+from allure_commons._allure import attach
+from reporting.reporting_utils import step
 
-## helper functions
+
+@step("Run \"file\" command")
```

Comment:
hm, so far the strip state checking is only a list of ssSSss on the console to reduce the noise (until errors occur) I wouldn't want it to take too much place in the report.

Reply:
If we do not add this to the report, we wouldn't be able to know whether we checked that files are stripped. It will not take to much visual space, because steps are collapsed by default.